content/docs/specs/native_histograms.md

```diff
@@ -0,0 +1,2416 @@
+
+Compared to the previously existing “classic” histograms, native histograms
+allow a higher bucket resolution across arbitrary ranges of observed values at
+a lower storage and query cost with very little to no configuration required.
+Even partitioning histograms by labels is now much more affordable.
+
+Because the sparse representation (property 1 in the list above) is so crucial
+for many of the other benefits of native histograms, _sparse histograms_ was
+a common name for _native histograms_ early during the design process. However,
+other key properties like the exponential bucketing schema or the dynamic
+nature of the buckets are also very important, but not caught at all in the
+term _sparse histograms_.
+
+### Design docs
+
+These are the design docs that guided the development of native histograms.
+Some details are obsolete now, but they describe rather well the underlying
+concepts and how they evolved.
+
+- [Sparse high-resolution histograms for
+  Prometheus](https://docs.google.com/document/d/1cLNv3aufPZb3fNfaJgdaRBZsInZKKIHo9E6HinJVbpM/edit),
+  the original design doc.
+- [Prometheus Sparse Histograms and
+  PromQL](https://docs.google.com/document/d/1ch6ru8GKg03N02jRjYriurt-CZqUVY09evPg6yKTA1s/edit),
+  more an exploratory document than a proper design doc about the handling of
+  native histograms in PromQL.
+
+### Conference talks
+
+A more approachable way of learning about native histograms is to watch
+conference talks, of which a selection is presented below. As an introduction,
+it might make sense to watch these talks and then return to this document to
+learn about all the details and technicalities.
+
+- [Secret History of Prometheus
+  Histograms](https://fosdem.org/2020/schedule/event/histograms/) about the
+  classic histograms and why Prometheus kept them for so long.
+- [Prometheus Histograms – Past, Present, and
+  Future](https://promcon.io/2019-munich/talks/prometheus-histograms-past-present-and-future/)
+  is the inaugural talk about the new approach that led to native histograms.
+- [Better Histograms for
+  Prometheus](https://www.youtube.com/watch?v=HG7uzON-IDM) explains why the
+  concepts work out in practice.
+- [Native Histograms in
+  Prometheus](https://promcon.io/2022-munich/talks/native-histograms-in-prometheus/)
+  presents and explains native histograms after the actual implementation.
+- [PromQL for Native
+  Histograms](https://promcon.io/2022-munich/talks/promql-for-native-histograms/)
+  explains the usage of native histograms in PromQL.
+- [Prometheus Native Histograms in
+  Production](https://www.youtube.com/watch?v=TgINvIK9SYc) provides an analysis
+  of performance and resource consumption.
+- [Using OpenTelemetry’s Exponential Histograms in
+  Prometheus](https://www.youtube.com/watch?v=W2_TpDcess8) covers the
+  interoperability with the OpenTelemetry.
+
+## Glossary
+
+- A __native histogram__ is an instance of the new complex sample type
```

Comment:
I think a native histogram is a native histogram of any schema. We should rather clarify at other locations (like above) rather than somehow suggesting that an NHCB is not a native histogram. (I tried to do that already, but as you have found, I wasn't entirely consistent.)

Reply:
I think we are on the same page then, thanks!